### ISSUE TITLE
Select's InfiniteScroll produces inaccurate screenreader results about the total number of options

### ISSUE THREAD

**Issue by phinch** [grommet/grommet#5898]
### Expected Behavior

When using a screenreader to navigate the Select input, I should be told the total number of options available.

### Actual Behavior

Instead, if the number of options exceeds `theme.select.step` in the theme object, the screenreader says that `select.step` options are available. The base Grommet theme sets step to 20, so for common use cases, like a select input that lists the 50 united states, the screenreader output is inaccurate. This is possible to circumvent by changing `select.step` for our needs, but it seems like it would be nice to make the input accessible by default, at least in more cases.

### URL, screen shot, or Codepen exhibiting the issue

https://codesandbox.io/s/grommet-v2-template-forked-53y0d?file=/index.js

### Steps to Reproduce

1. Use a screenreader to access the Select input and open the menu using Enter
2. Start navigating through the options by using the right arrow key
3. Observe that the screenreader reads out the options as "menu item [x] of 40" even though there are 50 items in the list

### Your Environment

- Grommet version: 2.19.0

**Comment by jcfilben** [comment]
Resolved in #5924
